crates/sui-indexer/src/backfill/backfill_instances/sql_backfills/event_sender.sh

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) Mysten Labs, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+sui-indexer --database-url postgres://postgres@localhost:5432/postgres run-back-fill "$1" "$2" sql "UPDATE events SET sender = CASE WHEN cardinality(senders) > 0 THEN senders[1] ELSE NULL END" checkpoint_sequence_number
```

Comment:
Should the cardinality always be 1 in our database today?

Reply:
in theory it should, checking if that's case.

Reply:
checked on both tnt and mnt db, all senders indeed always have exact 1 sender, will simplify the script